src/types/RequestResult.d.ts

```diff
@@ -15,5 +13,16 @@ export default class RequestResult {
     endTime: Date
   )
 
-  timeTaken: number
+  readonly method: string
+  readonly path: string
+  readonly query: object
```

Comment:
`object?` maybe?
URL query parameters. Only set if `method` is "GET".

Reply:
I just used the types from the `constructor` above shrug 🤷🏼‍♂️